app-text/htmltidy/tidy_git-5.7.0.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="The granddaddy of HTML tools, with support for modern standards"
+DESCRIPTION="Tidy tidies HTML and XML. It can tidy your documents by itself, \
+and developers can easily integrate its features into even more powerful tools."
+HOMEPAGE="http://www.html-tidy.org/"
+COPYRIGHT="1998-2015 World Wide Web Consortium"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="28be79db50d2ecf6730068167d8ed23987fa8d59"
+SOURCE_URI="https://github.com/htacg/tidy-html5/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="5fddac299a385203e06a2158df3acd68aa83ffd10d5bdf73b9e7f2913dee1a88"
```

Comment:
I think it would be better to rename the recipe to `tidy-5.7.0~git.recipe` (or `tidy57-5.7.0~git.recipe`).

Because with the current name (tidy_git-5.7.0.recipe) your PROVIDES are not 100% correct.
Besides, it is missing a CONFLICTS with the package for tidy-5.1.25.

If you wish to have both the old and the new, then you need to ajust the PROVIDES for the new package and add a CONFLICTS.

Anyway, it would be nice to insert a:
```
SOURCE_FILENAME="tidy-html5-$portVersion~$srcGitRev.tar.gz"
```
between CHECKSUM_SHA256 and SOURCE_DIR.

Reply:
I will revert this one to the one from 5.6.0 with the patches, this is actually a post 5.6.0/pre 5.7.0 :)

Reply:
Why don't you just drop the old recipe and add the new one as `tidy-5.7.0~git.recipe` ?
Then, when 5.7.0 is out, you will just need to rename the recipe to `tidy-5.7.0.recipe` and switch to the stable release tarball. Moreover, this method has the advantage that you don't need to worry about adding any CONFLICTS, and your PROVIDES are already OK.
Tidy pre-5.7.0 seems to be working fine, isn't it?
Nobody will complain if you remove the old recipe :)

Reply:
These changes will probably to into 5.6.1 as the developer mentioned, so maybe wait to update to the latest one after this one, so then we can leave out the patch, if nobody has a problem with it, I can still use the 5.7.0-pre version, but then when 5.6.1 commes out it wouldn't make any sence?

Reply:
Are we allowed to cheat here? I mean, put a `5.7.0~git` in `5.6.1~git` ?
Yeah, I know this is really bad.
Context:
`media-libs/libextractor/libextractor-1.3.recipe` is the **_only_** recipe which depends on `lib:libtidy` and it's not marked tested on any arch:
```
ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
SECONDARY_ARCHITECTURES="!x86_gcc2 ?x86"
```
(I don't want to push towards any direction. I'm just trying to see if we can avoid a patch, especially knowing that it will be removed when 5.6.1 becomes available.)

Another "solution" would be to leave this PR open for a few days or maybe one week, if nobody needs the new version of tidy right now.

Reply:
A bit confused here too ;)
We 'could' use a 5.6.1~git recipe that pull the 5.7.0~git sources
it would also mean changing the $libVersion though, as the ones created by the last one will be 5.7.0
Or I could leave this one in atm and add the one for 5.7.0~git to see where it goes

Choices ... ;)

Reply:
I think I know what's the best thing to do:

* we keep the old recipe and
* you add a `tidy57-5.7~git.recipe` with modifed PROVIDES, REQUIRES, and a CONFLICTS.

This will be better than adding a patch for 5.6.0, and  much better than pretending we have a `5.6.1~git` that would actually be a `5.7.0~git`. (Yes, my previous idea about cheating was really bad.)

Adding the CONFLICTS and changing the PROVIDES is not hard.
Just rename your recipe to `tidy57-5.7.0~git.recipe` and I'll show you what changes you need to make.

Reply:
tidy57-5.7~git.recipe ... can't seem to build this with haikuporter? 'not found in the repository'

Reply:
Changed to the new recipe, also changed the provide for the $libVersion, looking forward on the next step :)

Reply:
You forgot the ".0" in `tidy57-5.7.0~git.recipe` when you renamed your recipe.

Reply:
Oops, sorry @Begasus, I wrote `tidy57-5.7~git.recipe` instead of `tidy57-5.7.0~git.recipe`.
My mistake!